### PR TITLE
Release 2.27.0

### PR DESCRIPTION
**Pull request by moratom**
## Features
* Automatic crash dump collection when `DEPTHAI_ENABLE_FEEDBACK_CRASHDUMP` is set.
## Bug fixes
* Fix `PointCloud` generation when depth comes out of the `ImageAlign` node
* Fix IMX214 (OAK Lite) potential crash on init with more complex pipelines
* Fix a rare bug on a crash if a camera stream is closed after 1 second while AF is running

### CODE REVIEW

**Review by themarpe — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -27,6 +27,15 @@ To install PCL:
 MacOS: `brew install pcl`
 Linux: `sudo apt install libpcl-dev`
 
+> ℹ️ On Linux distributions based on RPMs, you need to install `perl-core` for us to OpenSSL dependency.
```
[the remark this answers is not in the document]
Comment (moratom) [reply README.md:30]: `perl-core` is a build time dependency for openssl, not runtime.

[the remark this answers is not in the document]
Comment (themarpe) [reply README.md:30]: Ah, got it. Thanks!

Then the suggested change should be:
```suggestion
> ℹ️ On Linux distributions based on RPMs, you need to install `perl-core` required by OpenSSL dependency.
```

File: src/utility/sha1.hpp [hunk 2]
```diff
@@ -0,0 +1,335 @@
+/*
```
[the remark this answers is not in the document]
Comment (themarpe) [reply src/utility/sha1.hpp:1]: Perhaps add a note for now, in the future we might be able to swap if needed

[the remark this answers is not in the document]
Comment (moratom) [reply src/utility/sha1.hpp:1]: On Windows, we don't use the OpenSSL but use `SCHANNEL` instead - so we don't have the `sha1` from there.

We could `ifdef` it to get it from the `SCHANNEL` that's used on Windows, but I figured it's easier to bring it in and have a single implementation.

[the remark this answers is not in the document]
Comment (moratom) [reply src/utility/sha1.hpp:1]: Added a comment that we can use SChannel and OpenSSL instead.

**Review by themarpe — APPROVED** [review]

**Pull request merged**